### PR TITLE
Spring app

### PR DESCRIPTION
**Pull request by volodymyrtaras**

### CODE REVIEW

File: .idea/vcs.xml [hunk 1]
```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (Anderwerty) [inline .idea/vcs.xml]: This file should not be in repository

File: src/main/java/App.java [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+
+public class App {
+    private static final String CONTEXT = "ApplicationContext.xml";
+    private static final String MODEL_ONE_FIRST_PROTOTYPE = "modelOneFirstPrototype";
+    private static final String MODEL_ONE_FIRST_SINGLETON = "modelOneFirstSingleton";
+    private static final String MODEL_ONE_SECOND_PROTOTYPE = "modelOneSecondPrototype";
+    private static final String MODEL_ONE_SECOND_SINGLETON = "modelOneSecondSingleton";
+
+    public static void main(String[] args) {
+        ModelOne modelOne = (ModelOne) new ClassPathXmlApplicationContext(CONTEXT)
```
Comment (Anderwerty) [inline src/main/java/App.java]: Why each time you create another context?
You should not.

File: src/main/java/implementations/ModelOne.java [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+
+@Slf4j
+@NoArgsConstructor
+@AllArgsConstructor
+@Data
+public class ModelOne implements Cellphone {
+    private Camera camera;
+    private Processor processor;
+    private Screen screen;
+
```
Comment (Anderwerty) [inline src/main/java/implementations/ModelOne.java]: Remove redundant blank line.

File: src/main/java/implementations/ModelOne.java [hunk 4]
```diff
@@ -0,0 +1,47 @@
...
+    public ModelOne(Camera camera, Processor processor, Screen screen) {
+        this.camera = camera;
+        this.processor = processor;
+        this.screen = screen;
+    }
+
+    @Override
+    public void produce() {
+        camera.showCamera();
```
Comment (Anderwerty) [inline src/main/java/implementations/ModelOne.java]: There is a problem with NPE. Please fix it.

File: src/main/resources/ApplicationContext.xml [hunk 5]
```diff
@@ -0,0 +1,62 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (Anderwerty) [inline src/main/resources/ApplicationContext.xml]: ApplicationContext.xml  ->applicationContext.xml 
Add better to configer context using annotation and java code but not xml.

**Comment by volodymyrtaras** [comment]
@Anderwerty fixed.

**Pull request merged**